test/getLockedTokens.js

```diff
@@ -1,9 +1,38 @@
 /* eslint-env mocha */
-/* global contract */
+/* global contract assert */
 
-contract('PLCRVoting', () => {
+const utils = require('./utils.js');
+
+contract('PLCRVoting', (accounts) => {
   describe('Function: getLockedTokens', () => {
-    it('should return the number of tokens the user has locked in polls');
+    const [alice] = accounts;
+
+    it('should return the number of tokens the user has locked in polls', async () => {
+      const plcr = await utils.getPLCRInstance();
+      const token = await utils.getERC20Token();
+      const options = utils.defaultOptions(); // 20 tokens
+      options.actor = alice;
+
+      const startingBalance = await token.balanceOf.call(alice);
+      const pollID = await utils.startPollAndCommitVote(options);
+      await utils.increaseTime(101);
```

Comment:
Rather than hardcoding this, grab the commit period from the config and do commitPeriod + 1 (using bignumbers, of course).